pages/cookbook/jettons.mdx

```diff
@@ -49,30 +100,41 @@ fun calculateJettonWalletAddress(ownerAddress: Address, jettonMasterAddress: Add
     return contractAddress(StateInit{code: jettonWalletCode, data: initData.toCell()});
 }
 
-contract Sample {
-    jettonWalletCode: Cell;
-    jettonMasterAddress: Address;
+message(0x7362d09c) JettonTransferNotification {
+    queryId: Int as uint64;
+    amount: Int as coins;
+    sender: Address;
+    forwardPayload: Slice as remaining;
+}
+
+contract Example with Deployable {
+    myJettonWalletAddress: Address;
+    myJettonAmount: Int as coins = 0;
 
     init(jettonWalletCode: Cell, jettonMasterAddress: Address) {
-        self.jettonWalletCode = jettonWalletCode;
-        self.jettonMasterAddress = jettonMasterAddress;
+        self.myJettonWalletAddress = calculateJettonWalletAddress(myAddress(), jettonMasterAddress, jettonWalletCode);
     }
 
     receive(msg: JettonTransferNotification) {
-        let myJettonWalletAddress = calculateJettonWalletAddress(myAddress(), self.jettonMasterAddress, self.jettonWalletCode);
-        require(sender() == myJettonWalletAddress, "Notification not from your jetton wallet!");
+        require(sender() == self.myJettonWalletAddress, "Notification not from your jetton wallet!");
+
+        self.myJettonAmount += msg.amount;
```

Comment:
We should also highlight that these notifications are not guaranteed to be sent. The default implementation doesn't send notification if `forwardAmount` is zero, so in such cases you can't really rely on them.